<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>581</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="chartLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>735</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboMethod">
         <item>
          <property name="text">
           <string>Интерполяционный полином Лагранжа</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Интерполяционный полином Ньютона</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Интерполяционный кубический сплайн</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>МНК: линейная регрессия</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>МНК: квадратичная регрессия</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>МНК: кубическая регрессия</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Интерполяционный тригонометрический полином</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitBtn">
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inDataValidationBtn">
         <property name="text">
          <string>Проверить данные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calculateBtn">
         <property name="text">
          <string>Вычислить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="idDataValidationLbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="chooseData"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="chooseData">
   <property name="text">
    <string>Выбрать файл с данными...</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
